woningwaardering/vera/bvg/model_uitbreidingen/eenheden_ruimte.py

```diff
@@ -35,3 +35,18 @@ class _EenhedenRuimte(BaseModel):
     """
     Geeft aan of de ruimte verwarmd wordt door een onroerende zaak. Dit wordt gebruikt bij het berekenen van de waardering van een ruimte.
     """
+    # https://github.com/Aedes-datastandaarden/vera-openapi/issues/63
+    breedte: Optional[float] = None
+    """
+    De maximale breedte van de ruimte gemeten in meters.
+    """
+    # https://github.com/Aedes-datastandaarden/vera-openapi/issues/63
+    lengte: Optional[float] = None
+    """
+    De maximale lengte van de ruimte gemeten in meters.
+    """
+    # https://github.com/Aedes-datastandaarden/vera-openapi/issues/63
+    hoogte: Optional[float] = None
+    """
+    De vrije hoogte van de ruimte gemeten in meters.
```

Comment:
```suggestion
    De maximale vrije hoogte van de ruimte gemeten in meters.
```

Willen we hier nog instructie geven dat bij geen plafond (buitenruimte) er `float('inf')` moet worden ingevuld? Of `None`?

Reply:
Onee 'infinity' bestaat niet in JSON he..